Makefile

```diff
@@ -352,7 +353,7 @@ verify:
 
 .PHONY: verify-modules
 verify-modules: modules
-	@if !(git diff --quiet HEAD -- go.sum go.mod hack/tools/go.mod hack/tools/go.sum); then \
+	@if !(git diff --quiet HEAD -- go.sum go.mod hack/tools/go.mod hack/tools/go.sum test/infrastructure/docker/go.sum); then \
```

Comment:
For followup, we should probably also consider doing similar against the test/infrastructure/docker/hack/tools directory also.